meson.build

```diff
@@ -1196,6 +1196,10 @@ else
     have_ea = false
 endif
 
+if host_os == 'darwin'
+    cdata.set('HAVE_EAFD', 1)
+endif
```

Comment:
There's a block starting on line 1120 with all of the platform specific conditions. Maybe you can merge these?